modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -325,6 +325,20 @@ CV__DNN_INLINE_NS_BEGIN
         static Ptr<PoolingLayerInt8> create(const LayerParams& params);
     };
 
+    class CV_EXPORTS ReduceLayer : public Layer
+    {
+    public:
+        int type;
```

Comment:
> type

`reduce_mode`? (to avoid confusion with data type)

Reply:
Thanks for your code review, all comments have been fixed.